docs/content/concepts/assets/asset-jobs.mdx

```diff
@@ -20,43 +20,43 @@ description: Asset jobs are the main unit for materializing and monitoring Softw
 
 ---
 
-## Relevant APIs
+## Creating asset jobs
```

Comment:
This is really a comment for the previous section, but I wonder if we can be a little more clear on the definition. It's a little confusing to talk about Jobs and then an asset job. 

Here's one attempt:

---

An asset job is a type of [Job](/concepts/ops-jobs-graphs/jobs] that targets a selection of software-defined assets which can be launched automatically via schedules or sensors, or manually from the UI.